scenes/game/game.gd

```diff
@@ -297,6 +298,8 @@ func setup_characters():
 			buddy.modulate = Color(1, 0.38, 0.55)
 	$PlayerZones/PlayerSealed.visible = 'has_sealed_area' in player_deck and player_deck['has_sealed_area']
 	$OpponentZones/OpponentSealed.visible = 'has_sealed_area' in opponent_deck and opponent_deck['has_sealed_area']
+	if 'sealed_area_is_secret' in opponent_deck and opponent_deck['sealed_area_is_secret']:
```

Comment:
Do you think you need the change from my Arakune branch in _on_add_to_sealed?
Otherwise it seems like the cards might show faceup briefly?

Reply:
Oh that's true; I'll borrow those changes